### PR TITLE
Remove inserting rarity calculations for nftMakeGenerative

### PR DESCRIPTION
**Pull request by elv-wayne**
Implement Sam's request to remove rarity calculations and use the provided attributes from the json files as is.
ie. Attributes without a "rarity" will be merged as is.

### CODE REVIEW

File: src/EluvioLive.js [hunk 1]
```diff
@@ -2344,13 +2332,12 @@ class EluvioLive {
       // Generative NFT - build an nft array
 
       // Read image and attributes info from directory
-      let { nftMetas, rarity } = await this.readNftDir({ nftDir });
+      let { nftMetas } = await this.readNftDir({ nftDir });
```
Comment (elv-serban) [inline src/EluvioLive.js:2335]: Can we still read this here in case we need it in the future?

File: src/EluvioLive.js [hunk 2]
```diff
@@ -2266,17 +2265,6 @@ class EluvioLive {
       },
     ];
 
-    // Insert rarity if doesn't exist
```
Comment (elv-serban) [inline src/EluvioLive.js:2269]: Is it useful to leave this code in here and just disable it?  Maybe make a constant 'insertRarity false' so we can turn on manually if needed in the future

File: src/EluvioLive.js (hunk 1 above)
Comment (elv-wayne) [reply src/EluvioLive.js]: Yes, it's still there.

File: src/EluvioLive.js (hunk 2 above)
Comment (elv-wayne) [reply src/EluvioLive.js:2269]: Latest commit has the insertRarity=false param and the code is left in there and only executes if true.

**Review by elv-serban — APPROVED** [review]

**Pull request merged**